.yarnrc.yml

```diff
@@ -1,10 +1,12 @@
+enableGlobalCache: true
+
 enableTransparentWorkspaces: false
 
 nodeLinker: node-modules
 
 plugins:
-  - path: .yarn/plugins/@yarnpkg/plugin-constraints.js
-    spec: "@yarnpkg/plugin-constraints"
+  - path: .yarn/plugins/@yarnpkg/plugin-deduplicate.cjs
+    spec: "https://raw.githubusercontent.com/eps1lon/yarn-plugin-deduplicate/latest/bin/%40yarnpkg/plugin-deduplicate.js"
```

Comment:
In the future we will remove this plugin and use built-in `yarn dedupe` ([PR](https://github.com/yarnpkg/berry/pull/1558)) which is expected to ship in Yarn 2.2.0.